src/cpp/core/include/core/json/JsonRpc.hpp

```diff
@@ -531,6 +531,41 @@ core::Error readObject(const json::Object& object,
    return Success() ;
 }
 
+inline core::Error readObject(const json::Object& object,
+                              const std::string& name,
+                              json::Array* pArray)
+{
+   return readObject<json::Array>(object, name, pArray);
+}
+
+template <typename T>
+core::Error readObject(
```

Comment:
This method (+ the one above) allow us to more easily read vectors of things, e.g. vectors of strings.